Taskfile.yml

```diff
@@ -45,7 +45,6 @@ tasks:
       - gofumpt -w .
       - go mod tidy
       - golangci-lint run --timeout 5m --fix
-      - nilaway -fix -test=false -include-pkgs='github.com/opslevel/kubectl-opslevel' ./...
```

Comment:
should not use -fix for this - too experimental. also remove nilaway on all pkgs since it makes CI super slow.

Reply:
Since we're still using `nilaway` in `task lint` and `task ci` by extension - how about just commenting this out?

Reply:
@davidbloss I am removing this line